includes/polling/os/hikvision-ds.inc.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$version  = trim(snmp_get($device, '.1.3.6.1.4.1.39165.1.3.0', '-OQv', 'HIK-DEVICE-MIB'), '"');
```

Comment:
You could use snmp_get_multi() to avoid multiple gets.
trim() is not needed.

Reply:
fixed.